functions/requests/routes/users.js

```diff
@@ -1,293 +1,35 @@
 const express = require('express');
 const admin = require('firebase-admin');
-const cors = require('cors')({ origin: true });
-const Busboy = require('busboy');
```

Comment:
If we aren't using anymore this libraries we should remove them from the package.json.